packages/terra-collapsible-menu-view/src/CollapsibleMenuViewItemGroup.jsx

```diff
@@ -78,11 +77,9 @@ class CollapsibleMenuViewItemGroup extends React.Component {
       );
     } if (isCollapsibleMenuItem) {
       return (
-        <li role="none">
-          <List {...customProps} role="listbox">
```

Comment:
Would this change break anything since it is no longer spreading the custom props on the list?

Reply:
As per this [comment](https://github.com/cerner/terra-framework/pull/946#discussion_r339127165) the `List` and `li`  are not providing any functionalities, but there could be a case where user might not be able to apply custom css classnames to the children.
cc @jmsv6d @benbcai

Reply:
I think we should be spreading the custom props on the div now

Reply:
Custom props added cb40173